test/models/location.test.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const Location = require('../../lib/models/location');
+
+describe('location model', () => {
+
+  describe('serialize', () => {
+
+    it('includes all of the necessary fields', () => {
+      const location = Location.forge().serialize();
```

Comment:
nit: use the `new` identifier instead of `.forge()`. See [this discussion](https://github.com/lob/lens-api/pull/212#discussion_r323993648).

Reply:
changed here and in the movie test as well